src/calculateRentalCost.js

```diff
@@ -28,7 +28,7 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return days < 3 ? days * 40 : days < 7 ? days * 40 - 20 : days * 40 - 50;
```

Comment:
Do not use nesting ternary operators - use if statements instead (without else or else if).
Also what does those magic numbers are? Create variable with respective names
Check [ckecklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md)